crates/theme/assets/dark/dark.ron

```diff
@@ -33,7 +33,8 @@ Theme (
                 "height": 36,
                 "foreground": "$LINK_WATER",
                 "icon_brush": "$LINK_WATER",
-                "background": "$LYNCH",
+                //"background": "$LYNCH",
+                "background": "linear-gradient(red, blue)",
```

Comment:
This was for testing purpose? Can you undo this please?

Reply:
Yeah, I was sure that I remove it